### ISSUE TITLE
hello, I don't know why I can't use it, could you help me ? thx!

### ISSUE THREAD

**Issue by andywu1998** [feng-lei/dl_coursera#1]
when I input
```
wuliweideMacBook-Air:~ wuliwei$ dl_coursera --email 107853xxxx@qq.com --password xxxx --slug os-pku --outdir ppij --how curl
```
the command return:
```
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.8/bin/dl_coursera", line 10, in <module>
    sys.exit(main())
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/dl_coursera_run.py", line 152, in main
    soc = crawl(args['email'], args['password'], args['slug'],
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/dl_coursera_run.py", line 45, in crawl
    soc = crawler.crawl(slug=slug, isSpec=isSpec)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/dl_coursera/Crawler.py", line 204, in crawl
    self.login()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/dl_coursera/Crawler.py", line 29, in login
    assert resp.status_code == 200
```

**Comment by feng-lei** [comment]
The issue has been fixed in `dl_coursera 0.1.1`. Since Coursera has changed its login API, you should not use `--email` and `--password` any more, use `--cookies` instead. Refer to **README** for more specific instructions.

**Comment by andywu1998** [comment]
thanks!
